services/device-registry/src/main/fabric8/deployment.yml

```diff
@@ -21,3 +21,5 @@ spec:
           value: dev
         - name: LOGGING_CONFIG
           value: classpath:logback-spring.xml
+        - name: HONO_REGISTRY_SVC_SAVE_TO_FILE
+          value: true
```

Comment:
Hi @ctron !

Did you notice that the credentials implementation has it's own flags and needs to enable persistence as well?

This has to be done by customizing the value of
`HONO_CREDENTIALS_SVC_SAVE_TO_FILE`.

Otherwise you get a device registry microservice that persists the registration entries, but not the credentials entries.
We discussed in the past if we should put all values in only one file, but since the two APIs are independant, this does not make too much sense, so we left them as two separate implementations in this service (with independent property values as well).

Reply:
No I didn't 😁 Thanks for pointing that out to me!

Reply:
I did amend the PR to add `HONO_CREDENTIALS_SVC_SAVE_TO_FILE` as well. Thanks @sysexcontrol